speech/cloud-client/beta_snippets.py

```diff
@@ -157,6 +158,35 @@ def transcribe_file_with_diarization(path):
 # [END speech_transcribe_diarization]
 
 
+# [START speech_transcribe_multichannel]
+def transcribe_file_with_multichannel(speech_file):
```

Comment:
I recommend showing an example value for `speech_file` within the snippet.

```
# [START speech_transcribe_multichannel]
from google.cloud import speech
client = speech.SpeechClient()

# TODO(developer): Uncomment and set to a path to your audio file.
# speech_file = 'path/to/file.wav'
...
```

Example:

https://github.com/GoogleCloudPlatform/google-cloud-python/blob/b022ca9d2176411c1d0a528da9ba820d24f0630e/docs/bigquery/snippets.py#L183